content/docs/user-guide/building/requirements.md

```diff
@@ -0,0 +1,121 @@
+# Requirements
+
+When you have a model saved via MLEM and want to use it, the first step is to
+make sure you have the right dependencies and packages. MLEM can get the list of
+requirements for you automatically for some type of requirements.
+
+## Pip based requirements
+
+`MLEM` can export the installable requirements needed for a model using the
+
+`mlem build requirements -m model`
+
+command, where `model` is the path to model saved via `mlem`
+
+```cli
+$ mlem build requirements -m model
+⏳️ Loading model from sk-model.mlem
+scikit-learn==1.0.2 pandas==1.4.2 numpy==1.22.3
+```
+
+Thus, the output of the above can be used in conjunction with:
+
+`pip install $(mlem -q build requirements -m model)`
+
+to install the requirements.
+
+where `-q` stands for the quiet mode which disables the emoji output.
```

Comment:
Oh, I see there was a 4th part as well :) Then 3 is better than 4 :)

Reply:
done